<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrderTab</class>
 <widget class="QWidget" name="CreateOrderTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
       <property name="text">
        <string>Order Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_order_name">
       <property name="minimumHeight">
        <number>35</number>
       </property>
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter order identifier or customer name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_employee">
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
       <property name="text">
        <string>Server / Employee:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_employee">
       <property name="minimumHeight">
        <number>35</number>
       </property>
       <property name="font">
        <font>
         <pointSize>12</pointSize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_items">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section { border: 1px solid #6c6c6c; }</string>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Qty</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_total_price">
       <property name="font">
        <font>
         <pointSize>14</pointSize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total: 0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="minimumHeight">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove_item">
       <property name="text">
        <string>Remove Selected Item</string>
       </property>
       <property name="minimumHeight">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_item">
       <property name="text">
        <string>Add Item</string>
       </property>
       <property name="minimumHeight">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_order">
       <property name="text">
        <string>Create Order</string>
       </property>
       <property name="minimumHeight">
        <number>40</number>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #4CAF50; color: white; font-weight: bold;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>